game_shared/ff/ff_projectile_incendiaryrocket.cpp

```diff
@@ -76,7 +90,18 @@ void CFFProjectileIncendiaryRocket::Explode(trace_t *pTrace, int bitsDamageType)
 		CFFPlayer *pPlayer = ToFFPlayer( pEntity );
 		if( g_pGameRules->FCanTakeDamage( pPlayer, pBurninator ) )

```

Comment:
Note that this does not do visibility checks, so the bonus damage will be able to travel through walls even if the direct damage didn't hit (a holdover from the TFC IC being able to light people on fire through walls).

Seems like this should use the same checks as RadiusDamage, be moved into RadiusDamage, or RadiusDamage should be refactored to be able to re-use the trace results outside of RadiusDamage.